conf/messages.en

```diff
@@ -465,26 +465,58 @@ pdf.ruling.aboutDecision.text=I have made this decision under section 24 of The
+
 pdf.ruling.section.askForReview.title=Asking for a review with HMRC
-pdf.ruling.section.askForReview.email=Email <strong>{0}</strong> asking for a review and give your reasons within 30 days of receiving your ruling email.
-pdf.ruling.section.ruling.details={0} details
+pdf.ruling.section.askForReview.reviewOfficer=A review officer, who is impartial, will look at the decision. The review outcome will be issued to you within 45 days.
+pdf.ruling.section.askForReview.ifYouDisagree=If you disagree with the outcome of the review, you can still appeal to the tribunal. You must do this within 30 days of the date of the letter telling you the outcome of the review. Once a review has commenced, you cannot appeal to a tribunal until the review officer tells you the outcome of their review.
+pdf.ruling.section.askForReview.furtherInfo=For further information about reviews, please see the following page.
+pdf.ruling.section.askForReview.guidance.url=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision</a>
+pdf.ruling.section.askForReview.requestReview=To request a review, please email, or write to, HMRC's Solicitors Office and Legal Services directly. You'll find the email and postal addresses below.
+pdf.ruling.section.askForReview.solsTeam=Because the SOLS team is a separate part of HMRC, they'll need the following information to identify your case and undertake the review:
+pdf.ruling.section.askForReview.solsTeam.1=your name, or business name, as shown on the top of this letter,
+pdf.ruling.section.askForReview.solsTeam.2=all references used at the top of this letter,
+pdf.ruling.section.askForReview.solsTeam.3=the name of the HMRC officer or team named at the end of this letter,
+pdf.ruling.section.askForReview.solsTeam.4=any new information,
+pdf.ruling.section.askForReview.solsTeam.5=reasons why you disagree with the decision.
+pdf.ruling.section.askForReview.solsTeam.cannotWrite=If there is a reason why you cannot write to the SOLS team within 30 days from the date of this letter, you will need a reasonable excuse for the delay to obtain an extension. You will need to write to the SOLS team as soon as possible to explain the delay in accepting the offer of a review. If you do not have a reasonable excuse, the review cannot be undertaken. You can ask a tribunal to accept a late appeal.
+pdf.ruling.section.askForReview.solsTeam.email=reviews@hmrc.gov.uk
+pdf.ruling.section.askForReview.solsTeam.emailText=The SOLS team email address is: <a class="govuk-link" href="mailto:{0}">{1}</a> If you decide to use email to correspond, please see the important information at the end of this letter. If you want to contact the team in writing the address is –
```

Comment:
Once`pdf.ruling.section.askForReview.solsTeam.email` has been removed, replace this `pdf.ruling.section.askForReview.solsTeam.emailText` with `pdf.ruling.section.askForReview.solsTeam.email`.

Also, hardcode the email `reviews@hmrc.gov.uk` in the message so it becomes: `The SOLS team email address is: <a class="govuk-link" href="mailto:reviews@hmrc.gov.uk">reviews@hmrc.gov.uk</a> If you decide to use email to correspond, please see the important information at the end of this letter. If you want to contact the team in writing the address is –` thanks